aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIO.java

```diff
@@ -65,6 +77,7 @@
   private transient S3Client client;
   private MetricsContext metrics = MetricsContext.nullMetrics();
   private final AtomicBoolean isResourceClosed = new AtomicBoolean(false);
+  private static volatile ExecutorService executorService;
```

Comment:
[question] Any reason, executor service need's to static ?

Reply:
nit: static variable should be placed before instance variables